src/components/Form/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Form extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {};
+	}
+
+	submitWrapper() {
```

Comment:
change to ```submitWrapper = () => {``` or bind member function into constructor with ```this.submitWrapper = this.submitWrapper.bind(this)```

Reply:
also can you rename submitWrapper to something that is more indicative of its functionality like ```handleSubmit```